plugins/rtmp-services/data/services.json

```
@@ -2808,6 +2808,75 @@
             "supported video codecs": [
                 "h264"
             ]
+        },
+        {
+            "name": "Livepeer Studio",
+            "more_info_link": "https://docs.livepeer.org/guides/developing/stream-via-obs",
+            "stream_key_link": "https://livepeer.studio/dashboard/streams",
+            "servers": [
+                {
+                    "name": "RTMP",
+                    "url": "rtmp://rtmp.livepeer.com/live"
+                }
```

Let's get these in there too, they're useful:

```suggestion
                {
                    "name": "Global Ingest",
                    "url": "rtmp://rtmp.livepeer.com/live"
                },
                {
                    "name": "Global Ingest (Primary)",
                    "url": "rtmp://rtmp-a.livepeer.com/live"
                },
                {
                    "name": "Global Ingest (Backup)",
                    "url": "rtmp://rtmp-b.livepeer.com/live"
                }
```

---

Added!